dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -842,7 +842,7 @@ CDefaultContextMenu::DoProperties(
     HRESULT hr = _DoCallback(DFM_INVOKECOMMAND, DFM_CMD_PROPERTIES, NULL);
 
     // We are asked to run the default property sheet
-    if (hr == S_FALSE)
+    if (hr == FALSE)
```

Comment:
`_DoCallback` returns an `HRESULT`, and `FALSE` is not an `HRESULT` so this is not a fix but a hack.

Reply:
`SH_ShowDriveProperties` is marked as returning a `BOOL`, but sometimes returns a `HRESULT`.
Then, `DrivesContextMenuCallback` handles this result as an `HRESULT`.
This needs to be fixed.